### PR TITLE
Fix location refresh issue

### PR DESCRIPTION
**Pull request by FabriceIRANKUNDA**
## Summary
Fixed error caused by updating user disconnected from the world
Passed instanceId to get all layer users

## Checklist
- [ ] Pre-push checks pass `npm run check`
  - [ ] Linter passing via `npm run lint`
  - [ ] Unit & Integration tests passing via `npm run test:packages`
  - [ ] Docker build process passing via `npm run build-client`
- [ ] If this PR is still a WIP, convert to a draft 
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewers

## References

_References to pertaining issue(s)_

## QA Steps

1. `git checkout pr_branch_name`
2. `npm install`
3. `npm run dev-reinit`
4. `npm run dev`

List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos.

## Reviewers

_Reviewers for this PR_

### CODE REVIEW

File: packages/engine/src/scene/functions/loaders/TransformFunctions.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const serializeTransform: ComponentSerializeFunction = (entity) => {
     name: SCENE_COMPONENT_TRANSFORM,
     props: {
       position: component.position,
-      rotation: new Vector3().setFromEuler(euler.setFromQuaternion(component.rotation)),
+      rotation: new Quaternion().setFromEuler(euler.setFromQuaternion(component.rotation)),
```
Comment (HexaField) [inline packages/engine/src/scene/functions/loaders/TransformFunctions.ts:49]: this rotations must be a vector3, not quaternion

File: packages/engine/src/scene/functions/loaders/PortalFunctions.ts [hunk 2]
```diff
@@ -105,7 +105,7 @@ export const serializePortal: ComponentSerializeFunction = (entity) => {
       redirect: portalComponent.redirect,
       // cubemapBakeId: component.cubemapBakeId, // TODO
       spawnPosition: helperTransform.position,
-      spawnRotation: new Vector3().setFromEuler(new Euler().setFromQuaternion(helperTransform.rotation))
+      spawnRotation: new Quaternion().setFromEuler(new Euler().setFromQuaternion(helperTransform.rotation))
```
Comment (HexaField) [inline packages/engine/src/scene/functions/loaders/PortalFunctions.ts:108]: this rotations must be a vector3, not quaternion

**Review by HexaField — CHANGES_REQUESTED** [review]

**Comment by HexaField** [comment]
Please `npm i` to fix the problem you are seeing with TransformFunctions and PortalFunctions, this is due to an out of date threejs version

File: packages/server-core/src/user/user/user.class.ts [hunk 3]
```diff
@@ -65,8 +65,8 @@ export class User<T = UserDataType> extends Service<T> {
       return super.find(params)
     } else if (action === 'layer-users') {
       delete params.query.action
-      params.query.instanceId = params.query.instanceId || loggedInUser.instanceId || 'intentionalBadId'
-      return super.find(params)
+      params.query.instanceId = params.query.instanceId
```
Comment (HexaField) [inline packages/server-core/src/user/user/user.class.ts:68]: why are we no longer falling back to `loggedInUser.instanceId` ?

Comment (FabriceIRANKUNDA) [reply packages/server-core/src/user/user/user.class.ts:68]: In some cases when user refresh the page  `loggedInUser.instanceId` is giving me a wrong instanceId.

**Review by HexaField — APPROVED** [review]

File: packages/client-core/src/user/services/UserService.ts [hunk 4]
```diff
@@ -109,18 +109,26 @@ export const UserService = {
...
 
-  getLayerUsers: async (instance = true) => {
+  getLayerUsers: async (instance) => {
     const dispatch = useDispatch()
-
-    const layerUsers = (await client.service('user').find({
-      query: {
-        $limit: 1000,
-        action: instance ? 'layer-users' : 'channel-users'
+    {
```
Comment (zulqarnainhanif) [inline packages/client-core/src/user/services/UserService.ts:114]: remove those unnecessary braces

File: packages/client-core/src/user/services/UserService.ts [hunk 5]
```diff
@@ -109,18 +109,26 @@ export const UserService = {
...
+        query: {
+          $limit: 1000,
+          action: instance ? 'layer-users' : 'channel-users',
+          instanceId
+        }
+      })) as Paginated<User>
+      dispatch(
+        instance ? UserAction.loadedLayerUsers(layerUsers.data) : UserAction.loadedChannelLayerUsers(layerUsers.data)
+      )
+    }
```
Comment (zulqarnainhanif) [inline packages/client-core/src/user/services/UserService.ts:131]: remove those unnecessary braces

**Review by zulqarnainhanif — CHANGES_REQUESTED** [review]
implement the 3 feedbacks left in comment.

**Review by zulqarnainhanif — APPROVED** [review]

**Pull request merged**